stable/cluster-autoscaler/charts/grafana/templates/k8s-autoscaler.yaml

```diff
@@ -58,7 +58,7 @@ data:
       "editable": true,
       "gnetId": 3831,
       "graphTooltip": 0,
-      "id": null,
+      "id": 36,
```

Comment:
this needs to be `null`

Reply:
i was wondering if that was going to have any effect. thanks, fixed in b19a677

Reply:
np, we ran into this a lot in dc/os when people started contributing dashboards - the trick is remembering to check the "export for sharing externally" box
![image](https://user-images.githubusercontent.com/5897740/85082007-7c730900-b182-11ea-8c93-bb4edd0e42d6.png)

Reply:
ty 🙇